### PR TITLE
Update RELEASE.md 

### PR DESCRIPTION
**Pull request by goldiegadde**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -12,14 +12,20 @@ TensorFlow 2.1 will be the last TF release supporting Python 2. Python 2 support
...
   * Automatic outside compilation is now enabled for Cloud TPUs. This allows `tf.summary` to be used more conveniently with Cloud TPUs.
   * Dynamic batch sizes with DistributionStrategy and Keras are supported on Cloud TPUs.
+  * Support for `.fit`, `.evaluate`, `.predict` on TPU using numpy data, in addition to `tf.data.Dataset`.
   * Keras reference implementations for many popular models are available in the TensorFlow [Model Garden](https://github.com/tensorflow/models/tree/master/official).
 * `tf.data`
   * Changes rebatching for `tf.data datasets` + DistributionStrategy for better performance. Note that the dataset also behaves slightly differently, in that the rebatched dataset cardinality will always be a multiple of the number of replicas.
+  * `tf.data.Dataset` now supports automatic data distribution and sharding in distributed environments, including on TPU pods.
+  * Distribution policies for `tf.data.Dataset` can now be tuned with 1. `data.experimental.AutoShardPolicy(OFF, AUTO, FILE, DATA)` 2. `data.experimental.ExternalStatePolicy(WARN, IGNORE, FAIL)`
```
Comment (rxsang) [inline RELEASE.md]: tf.data.experimental.AutoShardPolicy and tf.data.experimental.ExternalStatePolicy?

File: RELEASE.md [hunk 2]
```diff
@@ -12,14 +12,20 @@ TensorFlow 2.1 will be the last TF release supporting Python 2. Python 2 support
...
 * `tf.debugging`
   * Add `tf.debugging.enable_check_numerics()` and `tf.debugging.disable_check_numerics()` to help debugging the root causes of issues involving infinities and `NaN`s.
+* `tf.distribute`
+  * Custom training loop support on TPUs and TPU pods is avaiable through `strategy.experimental_distribute_dataset`, `strategy.experimental_run_v2`, `strategy.reduce`.
```
Comment (rxsang) [inline RELEASE.md]: Maybe add `strategy.experimental_distribute_datasets_from_function` as well.

**Review by rxsang — DISMISSED** [review]

File: RELEASE.md (hunk 1 above)
Comment (goldiegadde) [reply RELEASE.md]: Done.

File: RELEASE.md (hunk 2 above)
Comment (goldiegadde) [reply RELEASE.md]: Done.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**